### PR TITLE
[#4087] Omit history scanning on generated multiacc creation on deskt…

### PR DESCRIPTION
**Pull request by rasom**
Proposed by @saledjenic instead of making on client side.

status: ready

### CODE REVIEW

File: mobile/status.go [hunk 1]
```diff
@@ -351,6 +351,11 @@ func SaveAccountAndLogin(accountData, password, settingsJSON, configJSON, subacc
 	if err != nil {
 		return makeJSONResponse(err)
 	}
+
+	if *settings.Mnemonic != "" {
+		settings.OmitTransfersHistoryScan = true
```
Comment (saledjenic) [inline mobile/status.go:356]: @rasom could you point me to the piece of code where we're doing the history scan if new accounts are added on a later date, after the user backed up his mnemonic?

Comment (saledjenic) [reply mobile/status.go:356]: This change is ok, and that's the initial state when a user is creating a regular (non-keycard) Status account.
But then the user can back up the mnemonic, use the same seed phrase in other tools, then get back to Status and add the same account he used elsewhere. Do we handle that case?

Comment (rasom) [reply mobile/status.go:356]: We don't want to scan history right after account creation. It doesn't matter what happens with mnemonic afterwards in terms of this change.

Comment (saledjenic) [reply mobile/status.go:356]: Aha... so it's only for the default wallet account?

Comment (saledjenic) [reply mobile/status.go:356]: But anyway, I don't see code that avoids history scans based on `OmitTransfersHistoryScan` (not in this PR, but in `status-go` repo), do we have it or not yet?

**Review by saledjenic — APPROVED** [review]

File: mobile/status.go (hunk 1 above)
Comment (saledjenic) [reply mobile/status.go:356]: Ok, I see this https://github.com/status-im/status-go/pull/4163/files#diff-856ac71f99a073eb293c1213dc5ad2e9e620b0174fdc369cd39951d50a5117b4R106-R112 
thanks for sharing that

**Comment by qfrank** [comment]
Looks like mobile won't call `SaveAccountAndLogin` except `create-keycard-multiaccount` since mobile use new API `CreateAccountAndLogin` now, if you want to make this PR effect with mobile as well , proberbly additional changes are needed 🙂 @rasom

**Comment by rasom** [comment]
@qfrank it is covered for mobile in previous PR

**Review by qfrank — APPROVED** [review]
Oh, i see, there you go!

**Pull request merged**